connector/spanmetricsconnector/README.md

```diff
@@ -14,38 +14,29 @@
 
 ## Overview
```

Comment:
What are your thoughts around adding a link back to the original `spanmetrics` processor for posterity?

Reply:
Sure. I'll clean up and update the readme in a separate PR.